napari/layers/labels/labels.py

```diff
@@ -265,8 +268,8 @@ def selected_label(self):
 
     @selected_label.setter
     def selected_label(self, selected_label):
-        if selected_label < 0:
-            raise ValueError('cannot reduce selected label below 0')
+        if selected_label < 1:
```

Comment:
I think for now we should leave this as before as otherwise someone can't use the `fill` bucket with `0`, which is quite nice. I think though that we should keep your change of making the default `self._selected_label = 1`, this way if someone adds a new layer and starts painting they will see something right away